ui/js/controllers/perf/graphs.js

```diff
@@ -421,6 +422,38 @@ perf.controller('GraphsCtrl', [
                     updateDocument();
                     highlightDataPoints();
                 });
+
+                // This part of code is simply copy from overview-plot
+                $('#graph').bind("plotunselected", function() {
+                    $scope.zoom = {};
+                    $scope.selectedDataPoint = null;
+                    hideTooltip();
+                    updateDocument();
+                    plotGraph();
+                });
```

Comment:
This is identical to the overview graph plotunselected method.

You should define a function called plotUnselected somewhere up above and just bind it to both.

e.g. 
```
function plotUnselected() {
    $scope.zoom = {};
    $scope.selectedDataPoint = null;
    hideTooltip();
    updateDocument();
    plotGraph();
}

...

$("overview-plot").bind("plotunselected", plotUnselected);

...

$('#graph').bind("plotunselected", plotUnselected);
```